src/components/SimpleProfile/types.ts

```diff
@@ -12,6 +12,7 @@ export type GeneratedProfile = {
   referencePoints: ProfilePoint[];
   controlPoints: ProfilePoint[];
   curvePoints: Vec2[];
+  subdivisions: number[];
```

Comment:
Long profile segments should be subdivided after profile is scaled to the desired size / intensity. This is used to simplify mesh triangulation (all slices have the same number of points).